src/utilities/HYPRE_utilities.h

```diff
@@ -83,9 +83,44 @@ typedef double HYPRE_Real;
 #endif
 
 #if defined(HYPRE_COMPLEX)
+/* support for float double and long double complex types */
+#if defined(HYPRE_SINGLE)
+typedef float _Complex HYPRE_Complex;
+#define HYPRE_MPI_COMPLEX MPI_C_FLOAT_COMPLEX
+#ifdef HYPRE_SEQUENTIAL
+#define MPI_C_FLOAT_COMPLEX hypre_MPI_COMPLEX
```

Comment:
This should go in `mpistubs.h` (and hence also in `_hypre_utilities.h`).

Reply:
@rfalgout, just to be clear should the whole highlighted block be moved or just the new portion with HYPRE_SEQUENTIAL?

Reply:
All of the `MPI_` defines should be moved.  Sorry, I keep assuming that when I click on a line in github to make a comment, that the line is somehow highlighted.  I'll have to try to remember that's not the case.